scripts/run_lighthouse_tests.py

```diff
@@ -50,11 +62,143 @@ def run_lighthouse_checks():
         python_utils.PRINT(line[:-1])
 
 
-def main():
+def start_google_app_engine_server(prodEnv):
+    """Start the Google App Engine server."""
+    if prodEnv:
+        app_yaml_filepath = 'app.yaml'
+    else:
+        app_yaml_filepath = 'app_dev.yaml'
+
+    python_utils.PRINT('Starting oppia server...')
+
+    p = subprocess.Popen(
+        '%s %s/dev_appserver.py --host 0.0.0.0 --port %s '
+        '--clear_datastore=yes --dev_appserver_log_level=critical '
+        '--log_level=critical --skip_sdk_update_check=true %s' % (
+            common.CURRENT_PYTHON_BIN, common.GOOGLE_APP_ENGINE_HOME,
+            APP_ENGINE_PORT, app_yaml_filepath), shell=True)
+    RUNNING_PROCESSES.append(p)
+
+
+def download_and_install_nginx():
+    """Download and install nginx."""
+    python_utils.PRINT('Installing nginx...')
+    update_command = ['sudo', 'apt-get', 'update']
+    install_command = ['sudo', 'apt-get', 'install', 'nginx']
+
+    subprocess.check_call(update_command)
+    subprocess.check_call(install_command)
```

Comment:
What about other systems than Ubuntu, would it be possible to install `nginx` locally for Oppia?

Reply:
Hmm, okay, although Ubuntu is not one of the lightest systems I guess we have quite a lot experience using it and it makes sense to use it in th GitHub Actions.